axual-webclient/topic_config_data.go

```diff
@@ -9,6 +9,8 @@ type TopicConfigResponse struct {
 	ModifiedAt    string                 `json:"modified_at"`
 	CreatedBy     string                 `json:"created_by"`
 	ModifiedBy    string                 `json:"modified_by"`
```

Comment:
MINOR: The formatting of the whole object may need to change.

Reply:
I refactored it.